consensus/istanbul/validator/default.go

```diff
@@ -199,3 +204,13 @@ func (valSet *defaultSet) Copy() istanbul.ValidatorSet {
 func (valSet *defaultSet) F() int { return int(math.Ceil(float64(valSet.Size())/3)) - 1 }
 
 func (valSet *defaultSet) Policy() istanbul.ProposerPolicy { return valSet.policy }
+
+func (valSet *defaultSet) QuorumSize(formulaType uint8) int {
```

Comment:
Similar comment applies here. I think it's less error-prone if we don't give the caller the option to pick a formula. It's best if we remove the argument from the function and just stick to the int(math.Ceil(float64(2*valSet.Size())/3)) formula.